services/sentences.js

```diff
@@ -27,119 +22,85 @@ const GLOBAL_LOCATION = {
 
 export const adminSentences = {
   default:
-    'In 2010, {location} had {extent} of tree cover, extending over {percentage} of its land area.',
+    'In 2020, {location} had {extent} of tree cover, extending over {percentage} of its land area.',
   withLoss:
-    'In 2010, {location} had {extent} of tree cover, extending over {percentage} of its land area. In {year}, it lost {loss} of tree cover',
+    'In 2020, {location} had {extent} of tree cover, extending over {percentage} of its land area. In {year}, it lost {loss} of tree cover',
   globalInitial:
-    'In 2010, {location} had {extent} of tree cover, extending over {percentage} of its land area. In {year}, it lost {loss} of tree cover.',
+    'In 2020, {location} had {extent} of tree cover, extending over {percentage} of its land area. In {year}, it lost {loss} of tree cover.',
   withPlantationLoss:
-    'In 2010, {location} had {naturalForest} of natural forest, extending over {percentage} of its land area. In {year}, it lost {naturalLoss} of natural forest',
+    'In 2020, {location} had {naturalForest} of natural forest, extending over {percentage} of its land area. In {year}, it lost {naturalLoss} of natural forest',
   countrySpecific: {
     IDN: 'In 2001, {location} had {primaryForest} of primary forest*, extending over {percentagePrimaryForest} of its land area. In {year}, it lost {primaryLoss} of primary forest*, equivalent to {emissionsPrimary} of CO₂ emissions.',
   },
   co2Emissions: ', equivalent to {emissionsTreeCover} of CO\u2082 emissions.',
   end: '.',
 };
 
-export const getSentenceData = (params = GLOBAL_LOCATION) =>
-  all([
-    getExtent(params),
-    getExtent({ ...params, forestType: 'plantations' }),
-    getExtent({
+export const getSentenceData = async (params = GLOBAL_LOCATION) => {
+  try {
+    const extentNaturalForestResponse = await getExtentNaturalForest(params);
+    const lossNaturalForestResponse = await getLossNaturalForest({
       ...params,
-      forestType: 'primary_forest',
-      extentYear: 2000,
-    }),
-    getLoss(params),
-    getLoss({ ...params, forestType: 'plantations' }),
-    getLoss({ ...params, forestType: 'primary_forest' }),
-  ]).then(
```

Comment:
to this day I really don't know why this function is invoking 3 times `getExtent` and `getLoss` with different params,
also I ignore if they are used in a specific page besides the dashboard